app/docker/filters/filters.js

```diff
@@ -20,6 +20,12 @@ function hashToHexColor(hash) {
   return color;
 }
 
+function strToTimestamp(str) {
```

Comment:
I think that you can remove this function, it's not really used as a filter anyway (a filter is meant to be used in HTML).

Instead, you can relay on `moment` which is available as a library: https://momentjs.com/

See my comment below.

Reply:
Agree with you. Another question, how to import momentjs in this angularjs project? I guess there is place to include `moment` in yarn or npm or something?

Reply:
It's already added :-)

Simply using the suggestion I made should be fine (you'll want to test it tho)